src/components/search/detailed/AppSearchResults.js

```diff
@@ -195,16 +206,11 @@ export default function AppSearchResults(props) {
         return <Typography>{t("noResults")}</Typography>;
     }
 
-    let flatData = [];
-    if (data && data.length > 0) {
-        data.forEach((page) => {
-            flatData = [...flatData, ...page.apps];
-        });
-    }
```

Comment:
This method to get flatData on top of my changes was causing the drawer to infinitely render, so I moved this over into the `useEffect`